.eslintrc.js

```diff
@@ -214,10 +209,9 @@ module.exports = {
         },
     ],
     rules: {
-        "max-lines": "off",
```

Comment:
I removed this - unsure if that's a great idea.  There are a handful of files that trip the 1000 line limit (which is defined in our shared KA linter config). 

I think it's worth at least a warning if you go over 1,000 lines in a file. Anyone is free to eslint-ignore it, but at least it causes some pause to think about breaking up the file. 🤷‍♂️

Reply:
I guess 🤷‍♂️ Worth giving it a shot to see if anyone cares either way.